tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -231,11 +231,11 @@ toggle.
 ### <a id="downloading"></a>Downloading logs
 
 To download current log content, click the **Download logs** button.
+
 For optimal performance, the pod logs page limits the total log entries downloaded to the most recent
-10,000.
+10,000 (approximately).
```

Comment:
Same comment as above for usage of the word "approximately" here. See: https://github.com/pivotal/docs-tap/pull/1438/files#r948528694